gobblin-data-management/src/main/java/org/apache/gobblin/data/management/conversion/hive/materializer/HiveMaterializerFromEntityQueryGenerator.java

```diff
@@ -118,11 +119,19 @@ public QueryBasedHivePublishEntity generatePublishQueries() throws DataConversio
     Map<String, String> publishDirectories = publishEntity.getPublishDirectories();
     List<String> cleanupQueries = publishEntity.getCleanupQueries();
     List<String> cleanupDirectories = publishEntity.getCleanupDirectories();
+    Optional<Schema> avroSchema = Optional.absent();
```

Comment:
I might be wrong but is this method being called through the conversion jobs? By briefly looking at the code base, this `generatePublishQueries` method is only called in `org.apache.gobblin.data.management.conversion.hive.materializer.HiveMaterializer#generatePublishQueries` which is not related to Avro2ORC, just want to confirm.

Reply:
This is not called by Avro2Roc. I add this just to make sure it's consistent with that. But I can remove this one if needed

Reply:
You call. I am OK with both.